boot/freeldr/freeldr/ntldr/wlregistry.c

```diff
@@ -593,21 +609,112 @@ WinLdrScanRegistry(IN OUT PLIST_ENTRY BootDriverListHead,
                 rc = RegQueryValue(hDriverKey, L"Group", NULL, (PUCHAR)DriverGroup, &DriverGroupSize);
                 //TRACE_CH(REACTOS, "  Group: '%S'  \n", DriverGroup);
 
-                /* Make sure it should be started */
-                if ((StartValue == 0) &&
-                    (TagValue == OrderList[TagIndex]) &&
-                    (_wcsicmp(DriverGroup, GroupName) == 0))
+                /* Read the error control value */
```

Comment:
Why was the check removed?

Reply:
Because the redesign means that it now loads the entire contents of the Services key, rather than group by group.